src/include/distributed/connection_management.h

```diff
@@ -129,6 +129,7 @@ typedef struct ConnectionHashEntry
 {
 	ConnectionHashKey key;
 	dlist_head *connections;
+	int connectionCount;
```

Comment:
I guess we added this variable because there is no easy way of getting the length of `dlist`? Well, considering the gain we'd have as the connectionCount increases, I'd probably prefer to iterate over the list to count over adding a new field here :)